deno/sync-with-meilisearch/src/main.ts

```diff
@@ -0,0 +1,75 @@
+import { Client, Databases, Query } from "https://deno.land/x/appwrite@9.0.0/mod.ts";
+import { getStaticFile, interpolate, throwIfMissing } from './utils.ts';
+import { MeiliSearch } from "https://esm.sh/meilisearch@0.35.0";
+import { serve } from "https://deno.land/std@0.154.0/http/server.ts";
+
+const handler =  async (req : any, log : any) => {
+  throwIfMissing(Deno.env.toObject(), [
+    'APPWRITE_API_KEY',
+    'APPWRITE_DATABASE_ID',
+    'APPWRITE_COLLECTION_ID',
+    'MEILISEARCH_ENDPOINT',
+    'MEILISEARCH_INDEX_NAME',
+    'MEILISEARCH_ADMIN_API_KEY',
+    'MEILISEARCH_SEARCH_API_KEY',
+  ]);
+
+  if (req.method === 'GET') {
+    const html = interpolate(await getStaticFile('index.html'), {
+      MEILISEARCH_ENDPOINT: Deno.env.get('MEILISEARCH_ENDPOINT'),
+      MEILISEARCH_INDEX_NAME: Deno.env.get('MEILISEARCH_INDEX_NAME'),
+      MEILISEARCH_SEARCH_API_KEY: Deno.env.get('MEILISEARCH_SEARCH_API_KEY'),
+    });
+
+    return new Response(html,{ status: 200, headers: { 'Content-Type': 'text/html; charset=utf-8' } });
```

Comment:
Response is a deno object, we need to use OPR res object helper methods such as:
`res.send()`, `res.json()`